src/tutorials/composer-auth.md

```diff
@@ -0,0 +1,55 @@
+# Authenticated Composer repositories
+
+Some PHP projects may need to leverage a private, 3rd party Composer repository in addition to the public Packagist.org repository.  Often, such 3rd party repositories require authentication in order to download packages, and not everyone is comfortable putting those credentials into their Git repository source code (for obvious reasons).  To handle that situation, you can use [project variables]().
```

Comment:
Was pending on the rename decision.  I'll include it now on the assumption of the new name.